test/Microsoft.AspNet.Mvc.Core.Test/DefaultAssemblyProviderTests.cs

```diff
@@ -98,6 +99,46 @@ public void CandidateAssemblies_ReturnsEmptySequenceWhenReferenceAssembliesIsNul
             Assert.Empty(nullProviderCandidates.Select(a => a.Name));
         }
 
+        [Fact]
+        public void ReferenceAssemblies_ReturnsExpectedReferenceAssemblies()
```

Comment:
Needs some comments about what these tests are for, explain that you will want to modify the test if you've added a new dependency.